src/main/java/biz/paluch/logging/gelf/intern/sender/GelfKafkaSender.java

```diff
@@ -0,0 +1,48 @@
+package biz.paluch.logging.gelf.intern.sender;
+
+import biz.paluch.logging.gelf.intern.ErrorReporter;
+import biz.paluch.logging.gelf.intern.GelfMessage;
+import biz.paluch.logging.gelf.intern.GelfSender;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+
+import java.util.concurrent.Future;
+
+/**
+ * Gelf sender for kafka. This sender uses Kafka.
+ *
+ * @author Rifat Döver
+ * @since 1.13
+ */
+public class GelfKafkaSender implements GelfSender {
```

Comment:
Let's stick with `…GelfSender` naming scheme and rename the class to `KafkaGelfSender`

Reply:
Ok I did not know which one to choose because there was also _GelfHttpSender, GelfTCPSender, GelfUdpSender_, and _RedisGelfSender_. I have chosen heavily used one :)

Reply:
Also changing GelfKafkaSenderProvider to KafkaGelfSenderProvider